app/javascript/components/user/new-user.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div class="items-center justify-center w-full max-w-xs px-8 pt-6 pb-8 mb-4">
+    <h2 class="mb-4 text-2xl font-medium">
+      Nuevo Miembro
+    </h2>
+    <input
+      class="border rounded leading-tight mb-3 y-2 px-3 text-gray-700"
+      placeholder="Nombre"
+      type="text"
+      v-model="firstName"
+    >
+    <input
+      class="border rounded leading-tight mb-3 y-2 px-3 text-gray-700"
+      placeholder="Apellido"
+      type="text"
+      v-model="lastName"
+    >
+    <input
+      class="border rounded leading-tight mb-3 y-2 px-3 text-gray-700"
+      placeholder="Mail"
+      type="email"
+      v-model="email"
+    >
+    <button
+      class="flex items-center justify-center w-32 h-8 text-white bg-blue-800 rounded-2 hover:bg-blue-700"
+      @click="addUser"
+    >
+      Agregar
+    </button>
+    <div
+      class="flex flex-col items-center"
+    >
+      <loading v-if="loading" />
+    </div>
+  </div>
+</template>
+<script>
+import usersApi from '../../api/users';
+import Loading from '../shared/loading.vue';
+
+export default {
+  components: { Loading },
+  props: {
+    organizationId: {
+      type: String,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      firstName: '',
+      lastName: '',
+      email: '',
+      errors: [],
+      loading: false,
+    };
+  },
+  methods: {
+    userParams() {
+      const firstName = this.firstName;
+      const lastName = this.lastName;
+      const email = this.email;
+      const organizationId = this.organizationId;
+
+      return { email, firstName, lastName, organizationId };
+    },
+    async addUser() {
+      try {
+        this.loading = true;
+        const params = { user: this.userParams() };
+        await usersApi.createUser(params)
+          .then(() => {
+            window.location.replace('/organization');
+          });
```

Comment:
Aquí o el `then` o el `await` están de más, funcionaría solo con uno de los dos (independiente del caso). Yo prefiero `await`:

```suggestion
        await usersApi.createUser(params);
        window.location.replace('/organization')
```

Creo que es buena práctica no mezclar ambos! A veces puede ser medio confuso pal dev que no se maneja tanto en promesas cuando se mezcla.